CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Mapbox welcomes participation and contributions from everyone.
 * Remove `MapboxExperimental` annotation from viewport plugin and promote viewport plugin as stable API. ([1425](https://github.com/mapbox/mapbox-maps-android/pull/1425))
 * Introduce `addRendererSetupErrorListener`/`removeRendererSetupErrorListener` methods for `MapView` and `MapSurface` to listen to renderer setup errors and give opportunity to control some edge cases. ([1427](https://github.com/mapbox/mapbox-maps-android/pull/1427))
 * Introduce transition properties for atmosphere and terrain. ([1451](https://github.com/mapbox/mapbox-maps-android/pull/1451))
+* Enable main thread checking on the map/style object when running applications in debug mode. It's required to call these object functions on the main thread, otherwise you will hit edge case crashes. 
```

Comment:
```suggestion
* Enable main thread checking on the map/style object when running applications in debug build. It's required to call these object functions on the main thread, otherwise you will hit edge case crashes. 
```